server/middleware/smoketest/SmokeTestMiddleware.ts

```diff
@@ -0,0 +1,10 @@
+import express, { Router } from 'express'
+import SmokeTestStore from '../../data/cache/SmokeTestStore'
+
+export default function setupSmokeTest(smokeTestStore: SmokeTestStore): Router {
+  const router = express.Router()
+
+  router.post('/getSmokeTestSecret', async (req, res) => res.json({ token: await smokeTestStore.startSmokeTest(req) }))
```

Comment:
Re: my comment above .... ah, its called here .... this code could look quite complex if we implemented what I suggested ... Lets leave it for now